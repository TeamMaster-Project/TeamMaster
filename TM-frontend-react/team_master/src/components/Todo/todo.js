import React from 'react';

function Todo({}){
    return (
    <div data-testid="todo-1">
        <h1> Hellow</h1>
    </div>)
}

export default Todo;