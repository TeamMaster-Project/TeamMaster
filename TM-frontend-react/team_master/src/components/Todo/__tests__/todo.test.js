import React from 'react'
import {render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Todo from '../todo';

afterEach(()=>{
    cleanup();
})

test("Should Render Todo Component", () => { 
    // expect(true).toBe(true);
    render(<Todo/>);
    const todoElement = screen.getByTestId('todo-1');
    expect(todoElement).toBeInTheDocument();
    expect(todoElement).toHaveTextContent('Hello');
})

test('Matches Snapshot',()=>{
    const tree = renderer.create(<Todo/>).toJSON();
    // console.log(tree)
    expect(tree).toMatchSnapshot();
})