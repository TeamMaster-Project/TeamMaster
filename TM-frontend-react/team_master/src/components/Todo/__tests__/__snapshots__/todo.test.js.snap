// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Snapshot 1`] = `
<div
  data-testid="todo-1"
>
  <h1>
     Hellow
  </h1>
</div>
`;
